projects/demo/src/modules/components/editor/starter/editor-starter.template.html

```diff
@@ -6,6 +6,7 @@
     <ng-template pageTab="API">
         <tui-doc-demo [sticky]="false">
             <tui-editor
+                tuiAutoFocus
```

Comment:
I don't know if it is a significant issue but it does not work for IPhone.

https://user-images.githubusercontent.com/35179038/235081119-9da04435-80eb-4b00-b3cb-ce5a093be240.MP4

Reply:
(maybe we can create a separate issue with low priority to fix it later if there is no easy solution to do it in this PR)

Reply:
this needs to be configured at the level of the custom handler

with hack again:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/12021443/235089356-6d89eb92-fec7-410b-a2b1-306060a63547.png">